.github/workflows/deploy_website.yml

```diff
@@ -18,8 +18,7 @@ jobs:
       with:
         node-version: 16.13.2
     - run: npm ci
-    - run: npm run lerna -- bootstrap
```

Comment:
Edward added a `postinstall` script in `package.json` which  `npm` runs as part of its `npm install` or `npm ci` lifecycle. 
https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts

Because of this we removed all bootstrap steps in the GH pipelines

Reply:
Ok thanks, I missed this one